shell/platform/darwin/ios/framework/Source/FlutterViewControllerTest.mm

```diff
@@ -25,10 +26,6 @@ - (void)sendKeyEvent:(const FlutterKeyEvent&)event
             userData:(nullable void*)userData;
 @end
 
-namespace flutter {
-class PointerDataPacket {};
-}
```

Comment:
Did you mean to change this?

Reply:
Yes, if I didn't it wouldn't compile.

 I had to import
```
#import "flutter/lib/ui/window/platform_configuration.h"
``` 
to get access to the AccessibilityFeatureFlag enum.

platform_configuration imports the implementation of the PointerDataPacket class. Making the one in the tests obsolete.
```
#include "flutter/lib/ui/window/pointer_data_packet.h"
```